src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+            System.out.println("\t You now have " + tasks.size() + "tasks in the list.");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("\t Invalid task number!");
+        } catch (NumberFormatException e) {
+            System.out.println("\t Please enter the task number instead!");
+        }
+    }
+
+    private static void handleInput(String line) throws IllegalCommandWordException {
+        line = line.trim();
+        String[] wordsEntered = line.split(" ");
+        if (!COMMAND_WORDS.contains(wordsEntered[0].toLowerCase())) {
+            throw new IllegalCommandWordException();
+        } else if (line.toLowerCase().equals("list")){
+            listTasks();
+        } else if (line.toLowerCase().contains("done")){
+            markTaskAsDone(line);
+        } else if (line.toLowerCase().contains("delete")) {
+            // Insert code for deleting task
+            deleteTask(line);
+        } else {
+            try {
+                addTask(line);
+            } catch (MissingDeadlineException e) {
+                System.out.print("\tSorry, please indicate a valid deadline!\n");
+            } catch (IndexOutOfBoundsException e) {
+                System.out.print("\tSorry, please check if sufficient inputs are given.\n" +
+                        "\tTODO: details\n" +
+                        "\tDeadline: details, deadline\n" +
+                        "\tEvent: details, time of event\n");
+            } catch (MissingEventTimeException e) {
+                System.out.print("\tSorry, please indicate a valid event time!\n");
+            }
+        }
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
+    private static void setupTasks() throws FileNotFoundException,
+            MissingDeadlineException, MissingEventTimeException {
+        // Reads in current tasks when main() starts running.
+        File f = new File("duke.txt");
+        if (!f.exists()) {
+            try{
+                writeToFile("duke.txt", "");
+            } catch (IOException e) {
+                System.out.println("Something went wrong: " + e.getMessage());
+            }
+        } else {
+            // Reads in inputs from the file and update tasks
+            Scanner s = new Scanner(f);
+            while (s.hasNext()){
+                String line = s.nextLine();
+                String[] contents = line.split(" | ");
+                String taskDescription = contents[4];
+
+                switch (contents[0]){
+                case "T":{
```

Comment:
Using single characters to name cases might [hamper code readability](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short). See if a better name could be use to explain what the case is.